src/pages/spawn/spawnHost/fields/UserTagsField.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import { InstanceTag } from "gql/generated/types";
+import { convertArrayToObject } from "utils/array";
+import { UserTagRow } from "./userTagsField/UserTagRow";
+
+export interface UserTagsFieldStateType {
+  addedInstanceTags?: InstanceTag[];
+  deletedInstanceTags?: InstanceTag[];
+}
+
+interface UserTagsFieldProps {
+  data: UserTagsFieldStateType;
+  onChange: React.Dispatch<React.SetStateAction<any>>;
+  instanceTags: InstanceTag[];
+}
+export const UserTagsField: React.FC<UserTagsFieldProps> = ({
+  data,
+  onChange,
+  instanceTags,
+}) => {
+  const userTags = instanceTags.filter((tag) => tag.canBeModified);
```

Comment:
i get this `TypeError: Cannot read property 'filter' of null` when I open the edit modal with evergreen running on the master branch locally